src/analyze.ts

```diff
@@ -25,9 +25,9 @@ const asyncWalk: typeof import('../node_modules/estree-walker').asyncWalk = requ
 
 // Note: these should be deprecated over time as they ship in Acorn core
 const acorn = Parser.extend(
-  require("acorn-class-fields"),
-  require("acorn-static-class-features"),
-  require("acorn-private-class-elements")
+  //require("acorn-class-fields"),
```

Comment:
Should we remove these dependencies now? https://github.com/vercel/nft/blob/e89f3ca8cfa41838d0c4d7c16255eba7f561609f/package.json#L24-L26